solidity/random-datasource/randomExample.sol

```diff
@@ -59,6 +67,7 @@ contract RandomExample is usingOraclize {
         uint N = 7; // number of random bytes we want the datasource to return
         uint delay = 0; // number of seconds to wait before the execution takes place
         uint callbackGas = 200000; // amount of gas we want Oraclize to set for the callback function
+        emit LogNewOraclizeQuery("Oraclize query was sent, standing by for the answer...");
```

Comment:
(Didn't spot this before but oh well) Not that it matters because it's atomic, but it would make more sense to emit this _after_ the query, since it uses the past-tense but comes before the query...

Reply:
Approved, and also changed the same in the truffle-examples folder!